src/pybind/mgr/cephadm/templates/services/grafana/grafana.ini.j2

```diff
@@ -7,7 +7,7 @@
   org_role = 'Viewer'
 [server]
   domain = 'bootstrap.storage.lab'
-  protocol = https
+  protocol = {{ protocol }}
```

Comment:
this worked for the actual URL I could access grafana from in my browser (as in, that URL was http) but the URL the dashboard reports still reported https
```
[ceph: root@vm-00 /]# ceph orch ls --service-name grafana --export
service_type: grafana
service_name: grafana
placement:
  hosts:
  - vm-01
spec:
  protocol: http
[ceph: root@vm-00 /]# ceph config get mgr mgr/dashboard/GRAFANA_API_URL
https://vm-01:3000
```

Reply:
Fixed